### PR TITLE
[116] Read freaks from DB

### PR DESCRIPTION
**Pull request by gabrielgiroe1**
## Description
 Get data from DB and add  mock data from freak model

## Related
[[42] Read freaks from the DB](https://trello.com/c/tK6wOYCX/131-read-freaks-from-the-db)

## Steps to Test or Reproduce

Run the test from freak_create_spec.rb. Path: spec/controllers/graphql/mutations/freak_create_spec.rb

### CODE REVIEW

**Comment by alex-damian-negru** [comment]
Nice PR description. Please fill it in properly. How can one test this PR out?

**Review by alex-damian-negru — CHANGES_REQUESTED** [review]
Nice PR description. :eyes: 

Please fill it in properly. How can one test this PR out?

File: db/Guardfile [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
```
Comment (alex-damian-negru) [inline db/Guardfile:1]: This file doesn't belong here. Where did it come from?

File: spec/controllers/graphql/queries/freaks_spec.rb [hunk 2]
```diff
@@ -12,6 +12,22 @@ module Graphql
...
 
-    it { is_expected.to match_response_for(query: :Freaks, sample: :freaks) }
+    before do
+      create(:freak, name: 'Ion')
+      create(:freak, name: 'Gheorghe')
+    end
+
+    it { is_expected.to match_response_for(query: :freaks, sample: :freaks) }
+
+    context 'with pagination params' do
```
Comment (alex-damian-negru) [inline spec/controllers/graphql/queries/freaks_spec.rb:22]: Why are there pagination related specs here?

File: Gemfile [hunk 3]
```diff
@@ -37,11 +37,13 @@ gem 'graphql', '~> 1.12'
...
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot_rails', '~>6.2.0'
+  gem 'faker'
   gem 'rspec-rails', '~> 5.0.0'
 end
 
 group :development do
-  gem 'faker'
+  # gem 'faker'
```
Comment (alex-damian-negru) [inline Gemfile:46]: Why is this commented?

File: db/migrate/20210802074146_create_freaks.rb [hunk 4]
```diff
@@ -5,7 +5,7 @@ def change
     create_table :freaks do |t|
       t.string :name
       t.string :description
-      t.date :photo
+      t.string :photo
```
Comment (alex-damian-negru) [inline db/migrate/20210802074146_create_freaks.rb:8]: Why make this change here? Also, how does it relate with https://github.com/Agilefreaks/FreaksCatalog-API/pull/17?

File: spec/models/freak_spec.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Freak, type: :model do
+  context 'with validation test' do
+    let(:user) { build(:freak) } # user.new({})
+
+    it 'is ensures name present' do
+      build(:freak)
```
Comment (alex-damian-negru) [inline spec/models/freak_spec.rb:10]: The test does nothing. A `Freak` object is built and that's it. Please read https://www.justinweiss.com/articles/writing-better-tests-with-the-three-phase-pattern/ and apply it here.

**Review by alex-damian-negru — CHANGES_REQUESTED** [review]

**Comment by gabrielgiroe1** [comment]
[Read freaks from the DB](https://trello.com/c/tK6wOYCX/131-read-freaks-from-the-db)

File: Gemfile (hunk 3 above)
Comment (gabrielgiroe1) [reply Gemfile:46]: I forgot to delete the comment

File: spec/controllers/graphql/queries/freaks_spec.rb (hunk 2 above)
Comment (gabrielgiroe1) [reply spec/controllers/graphql/queries/freaks_spec.rb:22]: we unified the spec for the freaks node

**Pull request merged**